### ISSUE TITLE
JFIF files?

### ISSUE THREAD

**Issue by youngdarwin** [regosen/gallery_get#76]
I have never encountered a JFIF file, but GG doesn't seem to understand them. Here's a couple galleries:

https://www.imagefap.com/pictures/7767492/sexy-indian-girls
https://www.imagefap.com/pictures/8854175/Indian-Girls-7

In this case where there's one image on the page, for each image, GG reports: No links found at redirect page: http://www.imagefap.com/photo/2028476809/?pgid=&gid=7767492&page=1&idx=36

And then in it's summary, GG says: Errors were encountered!  Please check messages above.

**Comment by Hrxn** [comment]
JFIF = JPEG File Interchange Format

For all practical purposes, it's basically the same as a normal JPEG file (sans `.jfif` file extension, for example).

I assume gallery_get doesn't recognize this file extension, and henceforth can't find anything to process.

**Comment by youngdarwin** [comment]
Yes, I googled it. 

The points are (1) we should fix GG but (2) I think JFIF files are uncommon in the wild because this is the first time I've encountered this error. So maybe it's not super urgent. 

Also, I should have said that I'm renaming the files. I'm not sure if that has anything to do with getting the error.

I haven't yet tried examining the script to see where we'd have to modified it.
